src/main/kotlin/no/nav/familie/tilbake/dokumentbestilling/felles/task/PubliserJournalpostTask.kt

```diff
@@ -33,29 +36,48 @@ class PubliserJournalpostTask(
 
         val journalpostId = task.metadata.getProperty("journalpostId")
         val behandlingId = UUID.fromString(task.payload)
+
+        prøvDistribuerJournalPost(journalpostId, task, behandlingId)
+
+        val manuelleAdresser = manuellBrevmottakerService.hentBrevmottakereAsManuellAdresse(behandlingId)
+        manuelleAdresser.forEach { manuellAdresse ->
+            prøvDistribuerJournalPost(journalpostId, task, behandlingId, manuellAdresse)
+        }
+    }
+
+    private fun prøvDistribuerJournalPost(
```

Comment:
```suggestion
    private fun prøvDistribuerJournalpost(
```
Nitpick, men greit å holde det konsekvent gjennom koden 😛